src/components/ApplicationGroup/Constants.ts

```diff
@@ -1,4 +1,4 @@
-import { DeploymentNodeType } from '../app/details/triggerView/types'
+import { DeploymentNodeType } from '@devtron-labs/devtron-fe-common-lib'
```

Comment:
Please remove unused import

Reply:
This is used in title & some text consts